lambda/textractor/python/og.py

```diff
@@ -114,7 +122,7 @@ def _outputTable(self, page, p):
         S3Helper.writeCSVRaw(csvData, self.bucketName, opath)
         self.saveItem(self.documentId, "page-{}-Tables".format(p), opath)
 
-    def indexDocument(self, text, comprehendEntities):
+    def indexDocument(self, text, comprehendEntities, key_val_pairs):
```

Comment:
Since we are indexing more data, what are your thoughts about passing a dictionary with the keys for Entities & Pairs

Reply:
Can you please explain what you mean by passing a dictionary? both comprehendEntities and key_val_pairs are dictionaries only.